assets/src/features/leaving-page/LeavingRoomScreen.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from "react";
+import QuestionnairePageLayout from "./components/QuestionnairePageLayout";
+
+import Questionnaire from "./components/Questionnaire";
+import ReviewSentScreen from "./components/ReviewSentScreen";
+
+type LeavePageState = "questionnaire" | "review-sent";
+
+const LeavingRoomScreen: React.FC = () => {
+  const [pageState, setPageState] = useState<LeavePageState>("questionnaire");
+
+  return (
+    <QuestionnairePageLayout>
+      {pageState === "questionnaire" && <Questionnaire onSubmitClick={() => setPageState("review-sent")} />}
+      {pageState === "review-sent" && <ReviewSentScreen />}
+    </QuestionnairePageLayout>
```

Comment:
When refreshing the page after the review is sent, I'm getting back to the questionnaire. Is that intended? Would that be hard to change?

Reply:
The variable reflecting the leaving room state can be stored in a session or local storage of the user's browser. I'll think about that.

Reply:
It looks like this is not as trivial as I had initially thought. Can I turn this into a separate issue?

Reply:
[Jira ticket for this](https://membraneframework.atlassian.net/browse/MV-474)